src/schema/firefox-schemas-import.js

```diff
@@ -290,18 +377,38 @@ inner.loadSchema = (schema, file) => {
   return newSchema;
 };
 
-export function processSchemas(schemas, ourSchemas) {
+inner.mergeSchemas = (schemaLists) => {
+  const schemas = {};
+  Object.keys(schemaLists).forEach((namespace) => {
+    const namespaceSchemas = schemaLists[namespace];
+    if (namespaceSchemas.length === 1) {
+      schemas[namespace] = namespaceSchemas[0];
+    } else {
+      const file = `${namespace}.json`;
+      const merged = namespaceSchemas.reduce((memo, { schema }) => {
+        return merge(memo, schema);
+      }, {});
+      schemas[namespace] = { file, schema: merged };
+    }
+  });
+  return schemas;
+};
+
+export function processSchemas(schemas) {
   const loadedSchemas = {};
   schemas.forEach(({ file, schema }) => {
     // Convert the Firefox schema to more standard JSON schema.
     const loadedSchema = inner.loadSchema(schema, file);
-    loadedSchemas[loadedSchema.id] = { file, schema: loadedSchema };
+    const { id } = loadedSchema;
```

Comment:
It actually took me a while to notice the difference between loadedSchema and loaderSchema**s**, can we maybe call the ``id`` ``loadedSchemaId`` or at least ``loadedId`` or even work directly with ``loadedSchema.id`` instead?